app1React/package.json

```diff
@@ -21,10 +21,10 @@
   },
   "dependencies": {
     "prop-types": "^15.6.1",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
+    "react": "^16.4.1",
```

Comment:
the bug was only present with react@>=16.4.0 because single-spa-react@2.5.0 does some stuff with the new react context but that didn't exist before react@16.4.0